server/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "electric-store",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
add "start" script and use npm start for starting your app

Reply:
"start": "nodemone app.js"